app/views/coronavirus_form/business_details.html.erb

```diff
@@ -22,20 +22,39 @@
 <%= render "govuk_publishing_components/components/input", {
   id: "company_name",
   name: "company_name",
+  heading_size: "s",
   label: {
     text: t("coronavirus_form.questions.business_details.company_name.label"),
   },
   error_message: error_items('company_name'),
   value: @form_responses.dig(:business_details, :company_name),
 } %>
 
-<%= render "govuk_publishing_components/components/input", {
-  name: "company_number",
-  label: {
-    text: t("coronavirus_form.questions.business_details.company_number.label"),
-  },
-  error_message: error_items('company_number'),
-  value: @form_responses.dig(:business_details, :company_number),
+<%= render "govuk_publishing_components/components/radio", {
+  heading: t("coronavirus_form.questions.business_details.company_is_uk_registered.label"),
+  heading_size: "s",
+  name: "company_is_uk_registered",
+  error_message: error_items('company_is_uk_registered'),
+  items: t("coronavirus_form.questions.business_details.company_is_uk_registered.options").map.with_index do |(_, item), index|
+    {
+      id: ("company_is_uk_registered" if index == 0),
+      value: item[:label],
+      text: item[:label],
+      checked: @form_responses.dig(:business_details, :company_is_uk_registered) == item[:label],
+      conditional: item[:label] != t("coronavirus_form.questions.business_details.company_is_uk_registered.options.united_kingdom.label") ? nil :
```

Comment:
Or better yet, as there are only two items, rather than looping through them, perhaps they could added separately? This would remove the need for conditional logic around the company number input. We do something similar for [product details](https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/blob/master/app/views/coronavirus_form/product_details.html.erb#L81-L114).

Reply:
PENDING: Will take a look at this one...

Reply:
Done.  Thanks @leenagupte that looks much cleaner.  Good shout 👍